Feature/minimizer view (#1654)
seqan/seqan3: CHANGELOG.md

* The `seqan3::begin()`, `seqan3::end()`, `seqan3::cbegin()`, `seqan3::cend()`, `seqan3::size()`, `seqan3::empty()`
  functions have been deprecated:
  Use `std::ranges::{begin|end|cbegin|cend|size|empty}()` instead ([\#1663](https://github.com/seqan/seqan3/pull/1663)).

> * Added `seqan3::views::minimiser_view`, a view that minimisers for a range of comparable values given the
>   window_values_size

Reviewer: ```suggestion
* Added `seqan3::views::minimiser`, a view that computes the minimum in a window shifted over a range of comparable values.
```
